Japanese-sentence-submission.txt

```diff
@@ -0,0 +1,425 @@
+Instructions
+Please help us review the following sentences. These sentences were taken from a large source and certain words were modified automatically. On the right side you will see the most common issues we had with sentences. Please choose OK if this is a valid sentence in your language. If you encounter any of the issues you can select the letter (A, B, C, D) from the drop down menu. If you have any comments we are very grateful for them. Thank you for your help!
+
+
+
+
+
+
+
+Sentence
```

Comment:
These lines must be removed or else they will be imported as Japanese sentences :)